airqo-platform/AirQo-api
# Ft device registry thingspeak

- What does this PR do?
This PR creates 4 new endpoints for device registry.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NET-14).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/device-registry` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
**base URL**: http://localhost:3000/
[API doc ](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=897606332)for more details of this PR's endpoints below:
> **create Thing**: api/v1/devices/ts
> **delete Thing**: api/v1/devices/ts/delete?device="DEVICE ID"
> **clear Thing**: api/v1/devices/ts/clear?device="DEVICE ID"
> **update Thing**: api/v1/devices/ts/update?device="DEVICE ID"
